src/opower/utilities/enmax.py

```diff
@@ -0,0 +1,85 @@
+"""Enmax"""
+import aiohttp
+from typing import Optional
+import xml.etree.ElementTree as ET
+
+from ..const import USER_AGENT
+from ..exceptions import InvalidAuth
+from .base import UtilityBase
+
+class Enmax(UtilityBase):
+    @staticmethod
+    def name() -> str:
+        """Distinct recognizable name of the utility."""
+        return "Enmax Energy"
+
+    @staticmethod
+    def subdomain() -> str:
+        """Return the opower.com subdomain for this utility."""
+        return "enmx"
+
+    @staticmethod
+    def timezone() -> str:
+        """Return the timezone."""
+        return "America/Edmonton"
+
+    @staticmethod
+    def accepts_mfa() -> bool:
+        """Check if Utility implementations supports MFA."""
+        return False
+
+    @staticmethod
+    async def async_login(
+        session: aiohttp.ClientSession,
+        username: str,
+        password: str,
+        optional_mfa_secret: Optional[str],
+    ) -> None:
+
+        #Get request digest (required for authentication to Enmax)
+        async with session.post(
+            "https://www.enmax.com/SignInSite/_vti_bin/sites.asmx",
+            headers={
+                "User-Agent": USER_AGENT,
+                "Content-Type": 'text/xml',
+                },
+            data=b'<?xml version="1.0" encoding="utf-8"?><soap:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">  <soap:Body>    <GetUpdatedFormDigest xmlns="http://schemas.microsoft.com/sharepoint/soap/" />  </soap:Body></soap:Envelope>',
+            raise_for_status=True,
+        ) as resp:
+            xml = await resp.text()
+
+        xml = ET.fromstring(xml)
+        for i in xml.iter():
+            if(i.tag == "{http://schemas.microsoft.com/sharepoint/soap/}GetUpdatedFormDigestResult"):
```

Comment:
No parenthesis for conditions unless they are multiline. black will likely fix this for you.

Reply:
black didn't like the line length so it split it up